Ram NMR tutorial (#111)
castep-docs/castep-docs: docs/tutorials/NMR/Examples_3_-Alanine_and_Silicates.md

# Examples 3 - Alanine and Silicates

We now look at some more realistic and complex structures - the amino acid alanine and 2 different silicates: cristoballite and qartz. These are fairly large crystals - to get them
to complete in a short time we will run them on a cluster.
**Use the following number of
CPUs:**

* analine - 4
* cristoballite - 6
* quartz - 13

(these numbers are chosen to give efficient k-point scaling)

We will also perform EFG calculations for the above structures and examine the respective data.

> <p style="border-width:3px; border-style:solid;"><i>
> <b>Oxygen-17 NMR</b>
> 
> 
> Oxygen is a component of many geological materials. Oxygen is
> also important element in organic and biological molecules since it is often intimately involved in hydrogen bonding. Solid State <sup><small>17</small></sup>O NMR should be a uniquely valuable probe as the chemical shift range of <sup><small>17</small></sup>O covers almost 1000 ppm in organic molecules. Furthermore <sup><small>17</small></sup>O has spin I = 5/2 and hence a net quadrupole moment. As a consequence of this the solid state NMR spectrum is strongly affected by the electric
> field gradient at the nucleus.
> 
> Because the isotopic abundance of <sup><small>17</small></sup>O is very low (0.037%) and the NMR linewidths due to the electric field gradient relatively large, only limited Solid State NMR data is
> available. This is particularly true for organic materials. First principles calculations of <sup><small>17</small></sup>O NMR parameters have played a vital role in assigning experimental spectra, and developing empirical rules between NMR  parameters and local atomic structure.</i>
> </p>

Reviewer: Probably want Markdown not HTML formatting

```suggestion
### Oxygen-17 NMR

Oxygen is a component of many geological materials. Oxygen is
also important element in organic and biological molecules since it is often intimately involved in hydrogen bonding. Solid State ^17^O NMR should be a uniquely valuable probe as the chemical shift range of ^17^O covers almost 1000 ppm in organic molecules. Furthermore ^17^O has spin I = 5/2 and hence a net quadrupole moment. As a consequence of this the solid state NMR spectrum is strongly affected by the electric
field gradient at the nucleus.

Because the isotopic abundance of ^17^O is very low (0.037%) and the NMR linewidths due to the electric field gradient relatively large, only limited Solid State NMR data is
available. This is particularly true for organic materials. First principles calculations of ^17^O NMR parameters have played a vital role in assigning experimental spectra, and developing empirical rules between NMR  parameters and local atomic structure.
```

Author: In the original pdf this was in a black box so I'm not sure if it needs to be for any particular reason (and I don't think you can do that in markdown) - I'll change it to that for now but can always revert

> <h1>Examples 3 - Alanine and Silicates</h1>

Reviewer: Any reason not to use markdown heading here?

We now look at some more realistic examples. These are fairly large crystals - to get them
to complete in a short time we will run them on a cluster.
**Use the following number of
cpus:**

> **analine - 4 cristoballite - 6 quartz - 13**

Reviewer: Is this supposed to be a list?

<br>
Oxygen is a component of many geological materials. Oxygen is
also important element in organic and biological molecules since it is often intimately involved in hydrogen bonding. Solid State <sup><small>17</small></sup>O NMR should be a uniquely valuable probe as the chemical shift range of <sup><small>17</small></sup>O covers almost 1000 ppm in organic molecules. Furthermore <sup><small>17</small></sup>O has spin I = 5/2 and hence a net quadrupole moment. As a consequence of this the solid state NMR spectrum is strongly affected by the electric
field gradient at the nucleus.

Because the isotopic abundance of <sup><small>17</small></sup>O is very low (0.037%) and the NMR linewidths due to the electric field gradient relatively large, only limited Solid State NMR data is
available. This is particularly true for organic materials. First principles calculations of ^17^O NMR parameters have played a vital role in assigning experimental spectra, and developing empirical rules between NMR  parameters and local atomic structure.</i>
</p>

# Alanine

## Files

* [alanine.cell](alanine/alanine.cell)
* [alanine.param](alanine/alanine.param)
* [alanine.pdb](alanine/alanine.pdb)

> 

Reviewer: Are we wanting to host these files locally on the docs?

Author: The files (apart from the param ones) are too large to be displayed like the previous files - not sure how else we could give them people, and I think this is how some other documentation did it

Reviewer: That's fine, I just want to make sure. I can't resolve conversations on this repo, though.

<figure fig3>
  <figcaption>Fig3. Solid-State <sup><small>17</small></sup>O NMR spectrum of L-alanine. (b) is from MAS (magicangle- spinning) (c) is from DOR (double-orientation rotation)</figcaption>
</figure>

## Instructions

1. Look at the cell and param files. The geometry for alanine was obtained by neutron diffraction and was downloaded from the [Cambridge Crystallographic database](https://www.ccdc.cam.ac.uk/). View the original pdb file - you may use Materials Studio. Note the hydrogen bonding

2. Run the example - the calculation is not fully converged. However, the relative shift between the two sites is fairly converged.

3. The experimental ^17^O NMR spectrum shows two peaks (Fig 3 (b)) - they are very broad due to the quadrupolar coupling, and overlap. The experimental parameters are given in Table 1.

4. Assign the two resonances A and B. Do all three computed parameters support this assignment?

| | |
|--|--|
|&#948;(A)-&#948; (B) (ppm)| 23.5|
|C<sub>Q</sub> (A) (MHz)| 7.86|
|&#951;<sub>Q</sub> (A)| 0.28|
|C<sub>Q</sub>(B) (MHz)| 6.53|
|&#951;<sub>Q</sub>(B)| 0.70|

> | **Table 1: Experimental ^17^O NMR parameters for alanine. The two resonances are labeled A and B. Isotropic chemical shift &#948;, quadrupolar coupling C<sub>Q</sub>, and EFG asymmetry &#951;<sub>Q</sub>.**||

Reviewer: Suggest using tools to make these table columns align. I realise that markdown renders them fine, but It's quite useful if you have to change the source at some point. 

**N.B.** in emacs' `markdown-mode` from `MELPA` you can use `C-c <left>` - `C-c <right>` from within the table and it will swap columns right then left and correct the spacings as it does so.

Author: Is it emacs/MELPA/mark-down mode already installed wherever it needs to be? I'm trying to look this up but it's telling me I need to install it on my PC, but surely then it'll just work on my computer and not on the website?

Reviewer: Emacs is just the tool I use for editing the source which provides convenient tools for adding the "correct" spacing to the table, such that the columns are aligned visually in the markdown.
i.e.
```markdown
| table | header | 
| spa   | the    |
```
rather than
```markdown
| table | header | 
| spa | the |
```

Author: So I just use it locally and it'll give me how it should be written in markdown do you mean?

</figure>
 The experimental parameters are given in Table 1 below.

 |                             |     |
 |-----------------------------|-----|
 |$\delta$(A)-$\delta$(B) (ppm)| 23.5|
 |$C_Q$(A) (MHz)               | 7.86|
 |$\eta_Q$(A)                  | 0.28|
 |$C_Q$(B) (MHz)               | 6.53|
 |$\eta_Q$(B)                  | 0.70|

>  | **Table 1: Experimental ^17^O NMR parameters for alanine. The two resonances are labeled A and B. Isotropic chemical shift &#948;, quadrupolar coupling C<sub>Q</sub>, and EFG asymmetry $\eta_Q$.**||

Reviewer: May want latex maths for `&#948;` and `C<sub>Q</sub>`

```suggestion
| **Table 1: Experimental ^17^O NMR parameters for alanine. The two resonances are labelled A and B. Isotropic chemical shift $\delta$, quadrupolar coupling $C_{Q}$, and EFG asymmetry $\eta_Q$.**||
```

```

>  that the chemical shift alternates consistently between 48.29 and 62.86. To examine why this is the case we may look at the `alinine.cell`  file in Vesta (or the `alanine.pdb` file in Materials Studio, or any other option of your choice that shows hydrogen bonding). Similarly to [example 1](Example_1_-Ethanol.md), we will examine which atoms correspond to which environment.

Reviewer: ```suggestion
 that the chemical shift alternates consistently between 48.29 and 62.86. To examine why this is the case we may look at the `alanine.cell`  file in Vesta (or the `alanine.pdb` file in Materials Studio, or any other option of your choice that shows hydrogen bonding). Similarly to [example 1](Example_1_-Ethanol.md), we will examine which atoms correspond to which environment.
```
